mage_ai/server/app.py

```diff
@@ -110,12 +115,19 @@ def feature_set(id):
         }
     ]
     """
+    path = RESOURCE_PATH + '/FeatureSet'
```

Comment:
Let's add these logic into this class: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/server/data/base.py

Reply:
Added to base class